### PR TITLE
[REF] refactoring code, asynchronous render

### PR DESCRIPTION
**Pull request by GabbasovDinar**

### CODE REVIEW

**Comment by ilmir-k** [comment]
@GabbasovDinar fix lint errors

**Comment by GabbasovDinar** [comment]
@ilmir-k DONE

File: pos_print_method/doc/changelog.rst [hunk 1]
```diff
@@ -3,6 +3,11 @@
 Updates
 =======
 
+`1.0.1`
+-------
+
+- **IMP:** Refactoring the code to fix a slow sending of more product to the Kitchen (when using the separate printing method).
```
Comment (yelizariev) [inline pos_print_method/doc/changelog.rst]: many products

**Review by yelizariev — COMMENTED** [review]
Code is fine

**Review by ilmir-k — APPROVED** [review]

**Pull request merged**